README.md

```diff
@@ -1,2 +1,34 @@
 Ethereum On-Boarding Testsuite
-==============================
\ No newline at end of file
+==============================
+
+## Implement a Single Node Ethereum Test Network
+
+1. Create an account on https://kurtosistech.com if you don't have one yet.
```

Comment:
Let's make this `https://www.kurtosistech.com/sign-up` to direct them more specifically to where they need to go